packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -165,7 +165,7 @@ const SettingsSidebarContainer = ({
   return (
     <nav
       className={classNames(
-        "no-scrollbar fixed bottom-0 left-0 top-0 z-10 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll bg-gray-50 px-2 pb-3 transition-transform lg:sticky lg:flex",
+        "no-scrollbar fixed bottom-0 left-0 top-0 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll bg-gray-50 px-2 pb-3 transition-transform max-lg:z-10 lg:sticky lg:flex",
```

Comment:
what does max-lg:z-10 do?

Reply:
+1

Reply:
to fix a small issue with the sidebar in settings page
before:
![Screenshot 2023-01-22 131224](https://user-images.githubusercontent.com/37941426/213915233-2249563a-5599-4e46-baae-534d92275e92.png)
after:
![Screenshot 2023-01-22 131420](https://user-images.githubusercontent.com/37941426/213915244-1420d158-690f-4b1c-8ff2-dc0c7f649b77.png)
unless it was intended to appear that way 😅

Reply:
Excellent spot 🏆